### DISCUSSION TITLE
Ideas, Preguntas o Anotaciones de la reunión con Alicia 22-11-2024

### DISCUSSION THREAD

**Discussion by diegojiimenez** [TheMoys/24-25-IdSw1-SDR#41] [Q&A] [answered]
Aquí pondremos las preguntas o ideas que surjan durante o después de la reunión con Alicia.

**Comment by TheMoys** [comment, marked as answer]
- Cambiar categorías a grupos
- Cambiarlo a orden natural (Abrir, firmar, cerrar)
- Recordar que la confirmación de lectura y firmar es opcional
- La notificación debe enviarse a ambos usuarios (Remitente y destinatario)
- Cambiar responsable a usuario
- Tener en cuenta que el administrador también puede tener que firmar un documento
- El administrador puede tener un apartado de auditoría (“Diego a borrado una categoría”)
- Cambiar resumen a “DESCARGAR_INFORME_DEL_ENVIO”
